### ISSUE TITLE
The Miou output during training is inconsistent with the Miou tested？swin_ base model

### ISSUE THREAD

**Issue by daigang896** [open-mmlab/mmsegmentation#811]
The Miou output during training is inconsistent with the Miou tested. Upernet is used_ swin_ base_ patch4_ window12_ 512x512_ 160k_ ade20k_ pretrain_ 384x384_ 22K.py。 It uses its own data set. The validation set is the same 500 images and labels. The specific differences are as follows:

during train , val iou：
A   ： 99.53 
B   ： 72.25 
C   ： 72.63

after train , val iou：
A   :	99.26
B   :	51.73
C   :	55.77
I don't know what has such a great impact。when run，/mmsegmentation/mmseg/models/backbones/swin.py:611: UserWarning: DeprecationWarning: pretrained is a deprecated, please use "init_cfg" instead
  warnings.warn('DeprecationWarning: pretrained is a deprecated, '
I don't know if there's a problem. Hope to help.

**Comment by Junjun2016** [comment]
Could you please provide more information?
Try our official code.

**Comment by daigang896** [comment]
The official code is used. Single scale full image test. The Miou output during training is inconsistent with the Miou tested after train. config is upernet_swin_base_patch4_window12_512x512_160k_ade20k_pretrain_384x384_22K.py.
